cli/azd/pkg/project/project_config.go

```diff
@@ -37,7 +37,10 @@ type RequiredVersions struct {
 
 // options supported in azure.yaml
 type PipelineOptions struct {
-	Provider string `yaml:"provider"`
+	Provider                     string   `yaml:"provider"`
+	Variables                    []string `yaml:"variables"`
+	Secrets                      []string `yaml:"secrets"`
+	AdditionalVariablesAsSecrets bool     `yaml:"additionalVariablesAsSecrets"`
```

Comment:
`AdditionalVariablesAsSecrets` feels like a mode that changes the syncing behavior. 

The current name may be hard to understand in some cases. For example:

```yaml
pipeline:
   additionalVariablesAsSecrets: true
```

It's not immediately clear from reading this config, what the expected behavior is. I believe that this would result in all environment values set as secrets.

Reply:
yeah, I didn't get a perfect name, but it is exactly what you believed. It is a quick way to say `I want all .env values as secrets and I don't want to list them one by one`

I wanted to allow doing this only as `secrets` to cover scenarios where there is sensitive data on .env 

Any ideas for a better name?